phpunit/tests/fonts/font-library/wpFontFamily/getFontPost.php

```diff
@@ -22,7 +22,7 @@ public function test_should_return_post() {
 			'post_title'   => $this->merriweather['font_data']['name'],
 			'post_name'    => $this->merriweather['font_data']['slug'],
 			'post_type'    => 'wp_font_family',
-			'post_content' => '',
+			'post_content' => json_encode( $this->merriweather['font_data'] ),
```

Comment:
This test was failing because if `post_content` was empty — should this ever be allowed to be empty?

Reply:
I don't think the font post content should ever be allowed to be empty, otherwise that would mean we're missing the font data.